libs/src/sdk/config/staging.ts

```diff
@@ -5,13 +5,47 @@
 /* eslint-disable prettier/prettier */
 import type { ServicesConfig } from './types'
 export const servicesConfig: ServicesConfig = {
-  "minVersion": "0.3.66",
+  "minVersion": "0.3.83",
```

Comment:
Is this safe?

Reply:
def, we're way past that: https://healthz.audius.co/#/?isStaging=1